README.md

```diff
@@ -58,58 +76,34 @@ carthage update
 ```
 In your application targets “General” tab under the “Linked Frameworks and Libraries” section, drag and drop lottie-ios.framework from the Carthage/Build/iOS directory that `carthage update` produced.
 
-### Swift Package Manager
-``` swift
-// swift-tools-version:5.1
-
-import PackageDescription
-
-let package = Package(
-  name: "YourTestProject",
-  platforms: [
-       .iOS(.v12),
-  ],
-  dependencies: [
-    .package(name: "Lottie", url: "https://github.com/airbnb/lottie-ios.git", from: "3.2.1")
-  ],
-  targets: [
-    .target(name: "YourTestProject", dependencies: ["Lottie"])
-  ]
-)
-```
-And then import wherever needed: ```import Lottie```
-
-#### Adding it to an existent iOS Project via Swift Package Manager
-
-1. Using Xcode 11 go to File > Swift Packages > Add Package Dependency
-2. Paste the project URL: https://github.com/airbnb/lottie-ios
-3. Click on next and select the project target
-4. Don't forget to set `DEAD_CODE_STRIPPING = NO` in your `Build Settings` (https://bugs.swift.org/plugins/servlet/mobile#issue/SR-11564)
```

Comment:
I removed this because I have a hunch its not necessary anymore (although didn't test it specifically)